templates/.github/workflows/js-test-and-release.yml

```
@@ -124,7 +124,7 @@ jobs:
   release:
     needs: [test-node, test-chrome, test-chrome-webworker, test-firefox, test-firefox-webworker, test-electron-main, test-electron-renderer]
     runs-on: ubuntu-latest
-    if: github.event_name == 'push' && github.ref == 'refs/heads/${{{ github.default_branch }}}'
+    if: github.event_name == 'push' && github.ref == format('refs/heads/{0}', github.default_branch)
```

This one had `${{{}}}}` not `${{}}` 🤦 
```suggestion
    if: github.event_name == 'push' && github.ref == 'refs/heads/${{{ github.default_branch }}}'
```